data/apis/api3/apiMetadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "API3",
+  "active": true,
+  "homepage": "https://api3.org",
+  "airnode": "0x5b1243308f56156553F067D354b162f71E581b28",
+  "xpub": "xpub6C5CenwS58SBaET1q2FhY7jmBystDiBE5LDadX4P6MEAm11XDvnhPWYNXeuYtXBpWDbVZm5xbniiXMzWjur3nVVhms2naF8eEAwkdHnyrzp",
+  "logoPath": "https://www.amberdata.io/hubfs/amberdata_logo_color.svg",
+  "description": "API3 delivers efficient and secure first party oracle solutions.",
+  "maxSubscriptionPeriod": 3
+}
```

Comment:
I think we might want to also add back the data/apis/api3 nested folders that were removed in commit 688a528fcc5fac870bb709936514c5124bd4ca4d because create-boilerplate script is broken at the moment since it's looking for those files.

Reply:
I'll investigate, but `/data/apis/api3` isn't supposed to be a fixed folder (ie. it's another API provider, like Amberdata). We need to place test/third-party deployments in the api3-specific folder, rather than what we have now where we've stored a third party deployment in `/data/apis/amberdata` - so the script may be incorrect. Thanks again for pointing that out, I'll investigate :)

Reply:
and if you see anything else in the current scripts feel free to open a PR :)

Reply:
I think you might be referring to `/data/api3` as opposed to `/data/apis/api3`?
In the first case, it's an upper-level folder for api3-specific data (airseeker currently, from memory).
In the second case it's a 3rd-party (probably testing) deployment, in this case it's a third party deployment of Amberdata.

Reply:
I'm trying to follow the scripts from the start (to better understand the overall flow) and I'm stuck on the first one because of the mentioned error. The script is calling [this](https://github.com/api3dao/operations/blob/204e29846b974fd7344666bc7eacd706dcd2c2d1/src/utils/read-operations.ts#L20) function which is looking for the data/apis/api3 directory so you might be right in that the script might be out-of-date. It seems that the script is using those files as templates for generating a third party deployment.

Reply:
I've made a few changes in this branch that might resolve the issue, but I haven't expressly tested yet.

But yeah, nothing should assume/rely on the existence of `/data/apis/api3`.

Reply:
I'll be glad to test it. Also, how would you feel about having a `/data/apis/boilerplate` folder for the `create-boilerplate` script?

Reply:
@acenolaza you checkout https://github.com/api3dao/operations/pull/48 I think this would be a good fix

Reply:
I'll checkout Ashar's fix then 👍🏻